Source/WebCore/accessibility/AXTextRun.h

```diff
@@ -83,11 +84,19 @@ struct AXTextRuns {
     void* containingBlock { nullptr };
     Vector<AXTextRun> runs;
     bool containsOnlyASCII { true };
+    // A rough estimate of the size of the characters in these runs. This is per-AXTextRuns because
+    // AXTextRuns are associated with a single RenderText, which has the same style for all its runs.
+    // This is still an estimate, as characters can have vastly different sizes. We use this to serve
+    // APIs like AXBoundsForTextMarkerRangeAttribute quickly off the main-thread, and get more accurate
+    // sizes on-demand for runs that assistive technologies are actually requesting these APIs from.
+    static constexpr uint8_t defaultEstimatedCharacterWidth = 12;
+    uint8_t estimatedCharacterWidth { defaultEstimatedCharacterWidth };
```

Comment:
Most other classes use m_ for member variables, would make the code above easier to read by making it clear that estimatedCharacterWidth is an object variable rather than a local